mibs/hp/ENTITY-MIB

```diff
@@ -0,0 +1,1466 @@
+-- *****************************************************************
```

Comment:
Remove this mib please, it's already in the `mibs/` folder.

Reply:
@laf - Removed.

Reply:
@laf - Sorry, missed this one. Thanks for taking care of it.